lib/Position/Base91ParserUtils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Telemetry = require('./../MessageModels/Telemetry.js');
+
+module.exports = {
+    parseBase91Telemetry: (bodyContent) => {
+        //Extact and parse Base91 encoded telemetry from this spec: http://he.fi/doc/aprs-base91-comment-telemetry.txt
+        //This should be a sequence number, 1-5 TM values, and a character pair for 8 binary values (which can only appear if 5 other values are sent)
+        const msgString = bodyContent.match(/(.*)\|([!-{]{4,14})\|(.*)/);
+        if (msgString && msgString[2]) { //Found something. [2] is the TM data itself
+            const comment = bodyContent.replace('|' + msgString[2] + '|',''); //Remove the TM block from the comment. Need to add the pipes back. Regex stripped them.
```

Comment:
Nit, please add whitespaces after commas separating arguments in function calls

Reply:
Fixed